docs/spark/how-to-guides/broadcast-guide.md

```diff
@@ -12,8 +12,6 @@ In this article, you learn how to use broadcast variables in .NET for Apache Spa
 
 Because the data is sent only once, broadcast variables have performance benefits when compared to local variables that are shipped to the executors with each task. Refer to the [official broadcast variable documentation](https://spark.apache.org/docs/2.2.0/rdd-programming-guide.html#broadcast-variables) to get a deeper understanding of broadcast variables and why they are used.
 
```

Comment:
Why has this been removed? Let's modify it to remove the preview word but keep it in all guides as a link to our official Github repo.